### PR TITLE
MOL-79 Added custom refund amount to order overview.

### PR DESCRIPTION
**Pull request by iamreinder**

### CODE REVIEW

File: Controllers/Backend/MollieOrders.php [hunk 1]
```diff
@@ -334,14 +335,7 @@ public function refundAction()
...
 
-            # switch to the config and api key of the shop from the order
-            $shopSwitcher = new MollieShopSwitcher($this->container);
-            $this->config = $shopSwitcher->getConfig($order->getShop()->getId());
-            $this->apiClient = $shopSwitcher->getMollieApi($order->getShop()->getId());
-
-            $transaction = $this->orderService->getOrderTransactionByNumber($order->getNumber());
-
-            $refund = $this->refundService->refundFullOrder($order, $transaction);
+            $refund = $this->refundOrderFacade->refundOrder($order->getNumber(), $customAmount);
```
Comment (boxblinkracer) [inline Controllers/Backend/MollieOrders.php:338]: attention, you are removing the config and api switching!
in the backend, orders can come from any subshop...so this is always related to the provided order that will be refunded. i think this breaks things

**Comment by boxblinkracer** [comment]
good job @iamreinder 
i would merge it

could you arrange the modal a bit before. 
the main (german) market has a truncated text...maybe you can increase the height, or even better, increase column1 a little bit. then it would all fit in 1 line

thank you

Screen Shot 2022-09-19 at 09 22 49

**Comment by iamreinder** [comment]
@boxblinkracer I'm fixing the window right now, this is how it will look:

image

**Comment by boxblinkracer** [comment]
hi @iamreinder 
awesome thanks
its still not completely visible (see textfield)
but i will just adjust that after the merge
thanks

**Pull request merged**